tools/buildsteps/windows/arm-uwp/BuildSetup.bat

```diff
@@ -0,0 +1,12 @@
+@ECHO OFF
+
+SET cmakeGenerator=Visual Studio 14 ARM
+SET TARGET_ARCHITECTURE=x86
```

Comment:
ya, rly. it doesn't matter because `TARGET_ARCHITECTURE` will not be used for uwp packaging.